app/controllers/mixins/actions/provider_actions/mass_transform.rb

```diff
@@ -0,0 +1,56 @@
+module Mixins
+  module Actions
+    module ProviderActions
+      module MassTransform
+        def vm_transform_mass
+          dialog = Dialog.find_by(:label => 'Transform VM')
```

Comment:
@smelamud Should this be find_by! so that it raises an error if the Dialog is not found.